keras_trainer/callbacks.py

```diff
@@ -0,0 +1,49 @@
+import os
+import numpy as np
+
+from keras.callbacks import Callback
+from sklearn.metrics import recall_score
+
+
+class SensitivityCallback(Callback):
```

Comment:
One thought is that this class could be more general, something like a `MetricCallback()`, since little of this code is specific to sensitivity! The class could take as input a metric and a string describing the metric to really compute any metric we like! Would be a nice addition and allow us to experiment with other ideas!

Reply:
I agree with that! I just thought it would be simpler that way to implement it inside `keras-trainer` as I was re-using other parts of that code (data generator). It would mean that this metrics will have to be defined somewhere (Triage-net)...
It could be a nice addition in the future!